src/js/constants/ServiceSchema.js

```diff
@@ -81,6 +81,63 @@ let SERVICE_SCHEMA = {
         }
       },
       required: []
+    },
+    'Optional': {
```

Comment:
Oh, I totally agree. But that's how the SchemaForm works right now and I think refactoring it to add a `sectionId` or similar is out of scope.